src/m365/purview/commands/retentionevent/retentionevent-list.ts

```diff
@@ -0,0 +1,30 @@
+import { Logger } from '../../../../cli/Logger';
+import { odata } from '../../../../utils/odata';
+import GraphCommand from '../../../base/GraphCommand';
+import commands from '../../commands';
+
+class PurviewRetentionEventListCommand extends GraphCommand {
+  public get name(): string {
+    return commands.RETENTIONEVENT_LIST;
+  }
+
+  public get description(): string {
+    return 'Get a list of retention events';
+  }
+
+  public defaultProperties(): string[] | undefined {
+    return ['id', 'displayName', 'eventTriggerDateTime'];
+  }
+
+  public async commandAction(logger: Logger): Promise<void> {
+    try {
```

Comment:
Could you add a verbose logging line along the lines of 'Retrieving Purview retention events' and can you add a check for delegated permissions and throw an error if it is running app only?